pytest_perf/plugin.py

```diff
@@ -14,11 +14,37 @@
 from pytest_perf import runner
 
 
-def pytest_collect_file(parent, path):
+def pytest_addoption(parser):
+    group = parser.getgroup("performance tests (pytest-perf)")
+    group.addoption(
+        "--perf-target",
+        dest="perf_target",
+        default=".",
+        action="store",
+        help="directory or distribution file for which the performance tests will run"
+        " (default: %(default)r)",
+    )
+    group.addoption(
+        "--perf-baseline",
```

Comment:
What's the use-case for this option? It feels a little like feature creep, but I'm open to explore it.

Reply:
My main use case is to be able to run tests in the development machine:

Usually contributors will fork the repository and therefore the `origin` upstream will not correspond to the original project. For quick fixes and updates it even is common to just do it in the main branch of the contributor's fork and file a simple PR.

For example, in this project, I am using:

```bash
tox -- --perf-baseline https://github.com/jaraco/pytest-perf
```

otherwise  the performance test run against `git@github.com:abravalheri/pytest-perf`.

That said, this CLI option is just a minor convenience that I am more than OK in dropping.

I just added the CLI by mirroring the new parameters of the `BenchmarkRunner` class, and I added a `baseline` parameter there to be able to write some tests.

Reply:
> Usually contributors will fork the repository and therefore the origin remote will not correspond to the original project. For quick fixes and updates it even is common to just change the main branch of the contributor's fork and open a simple PR.

I would think this plugin would still function in that case, as it would compare the performance of the local checkout against their origin. Of course, if they push those changes, the performance tests no longer reflect the difference... but then they would again when a pull request is made.

I do think it may be worthwhile giving the user this ability to customize, but given my concerns about the implementation, I think I'd like to address this concern separately, perhaps first if it's a prerequisite for the wheel support.